src/coreclr/vm/codeman.cpp

```diff
@@ -1525,7 +1525,16 @@ void EEJitManager::SetCpuInfo()
 
     if (((cpuFeatures & ARM64IntrinsicConstants_Sve) != 0) && CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_EnableArm64Sve))
     {
-        CPUCompileFlags.Set(InstructionSet_Sve);
+        int maxVectorLength = (maxVectorTBitWidth >> 3);
+        uint64_t systemVectorTLength = GetSystemVectorLength();
```

Comment:
nit: This name is potentially confusing since we have the `System` namespace and many concepts related to `Vector`.

Something that explicitly says `Sve` in the name would be much clearer IMO. Perhaps `GetSveLengthFromOS()` or `GetSveLengthFromSystem()` instead, just to help avoid ambiguities here?

Reply:
Done